index.html

```diff
@@ -40,6 +40,7 @@
 
         <hr>
 
+        <input type="search" class="light-table-filter" data-table="sortable" placeholder="Filter" size="100">
```

Comment:
Oh also `class="light-table-filter form-control"` so that it uses the bootstrap css.